en/docs/learn/consent-management-with-single-sign-on.md

```diff
@@ -79,10 +79,10 @@ authentication.
         [Consent management](../../learn/user-portal/#consent-management)
         .
 
-    -   The application requires new mandatory claims that the user has
-        not consented to previously. If the service provider requests
-        for any new mandatory claim values, the user will only be
-        prompted to provide consent for the newly added mandatory claims
+    -   The application requires new claim that the user has
+        not consented previously. If the service provider requests
+        for any  claim values, the user will only be prompted to provide 
```

Comment:
```suggestion
        not consented previously. If the service provider is configured to request
        for any  claim values, the user will only be prompted to provide 
```

WDYT?

Reply:
Additional space before "  ****claim values" can be removed.

Reply:
Fixed https://github.com/wso2/docs-is/pull/2485/commits/9c146abfa71e33920dee62e602ac38868c387825